src/components/products/Products.jsx

```diff
@@ -1,31 +1,27 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 import Product from "./Product";
 
-import {
-    Pane,
-  } from "evergreen-ui";
+import { Pane } from "evergreen-ui";
 import "../../assets/css/products/product.css";
-  
+
 export default class Products extends Component {
-   
-    
-    render() {
-        return (
-            <Pane className="products">
-                {this.props.products.map((product)=>(
-                    <Product 
-                    key={product.id} 
-                    name={product.name}
-                    type={product.type}
-                    tag={product.tag}
-                    img={product.img}
-                    color={product.color}
-                    size={product.size}
-                    price={product.price}
-                    />
-                ))}
-                
-            </Pane>
-        )
-    }
+  render() {
+    const productclass = this.props.shownav ? "products" : "products-nonav";
```

Comment:
why is this variable necessary here? it seems whatever the value is set for the variable, the contents of the div remains the same for both the value of variable

Reply:
no the contents are not the same if you have actually executed the program you will see that products which triggers the showfilter has width of 84% and products-nonav have a 100% width which is triggered upon Hidefilter button.